src/Android/Receivers/NotificationDismissReceiver.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Android.Content;
+using Bit.App.Abstractions;
+using Bit.App.Models;
+using Bit.App.Services;
+using Bit.Core;
+using Bit.Core.Abstractions;
+using Bit.Core.Services;
+using Bit.Core.Utilities;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using static AndroidX.Concurrent.Futures.CallbackToFutureAdapter;
+
+namespace Bit.Droid.Receivers
+{
+    [BroadcastReceiver(Name = "com.x8bit.bitwarden.NotificationDismissReceiver", Exported = false)]
```

Comment:
Change this to use the new `Constants.PACKAGE_NAME` prefix and if possible use nameof:
```c#
[BroadcastReceiver(Name = Constants.PACKAGE_NAME + "." + nameof(NotificationDismissReceiver), Exported = false)]
```